upgrade/install-3.0.php

```diff
@@ -79,5 +79,15 @@ function upgrade_module_3_0($object, $install = false)
 			) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 AUTO_INCREMENT=1');
 	}
 
+	//Add datas to paypal order
+	Db::getInstance()->Execute('
```

Comment:
Hi @Quetzacoalt91 ,

Thanks for the advice and examples. 
I've just modified the behavior to use a new upgrade file.